cos-fleet-catalog-connectors/gcp/google-bigquery-0.1/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+    <parent>
+        <groupId>org.bf2</groupId>
+        <artifactId>cos-fleet-catalog-connectors-gcp</artifactId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>cos-connector-google-bigquery-0.1</artifactId>
+
+    <name>Google BigQuery</name>
+    <description>Google BigQuery</description>
+
+    <packaging>jar</packaging>
+
+    <properties>
+        <cos.connector.type>google-bigquery</cos.connector.type>
+        <quarkus.container-image.name>cos-connector-${cos.connector.type}</quarkus.container-image.name>
+        <quarkus.container-image.tag>${cos.connector.version}.${cos.connector.revision}</quarkus.container-image.tag>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.bf2</groupId>
+                <artifactId>camel-connector-maven-plugin</artifactId>
+                <configuration>
+                    <connectors>
+                        <connector>
+                            <name>${cos.connector.type}-sink-${cos.connector.version}</name>
+                            <title>${project.name} sink</title>
+                            <description>${project.description} sink</description>
+                            <adapter>
+                                <prefix>gcp</prefix>
+                                <name>${cos.connector.type}-sink</name>
+                                <version>${camel-kamelets-catalog.version}</version>
+                            </adapter>
+                            <kafka>
+                                <prefix>kafka</prefix>
+                                <name>cos-kafka-source</name>
+                                <version>${cos.kamelets.version}</version>
+                            </kafka>
+                            <dataShape>
+                                <consumes>
+                                    <formats>
+                                        <format>application/octet-stream</format>
```

Comment:
this need to be confirmed as the related kamelet, declares it consumes json https://github.com/apache/camel-kamelets/blob/main/kamelets/google-bigquery-sink.kamelet.yaml#L64-L65